admin/services/data-access/pupil.data.service.js

```diff
@@ -163,9 +163,12 @@ pupilDataService.sqlFindPupilsWithActivePins = async (dfeNumber) => {
   winston.debug(`sqlFindPupilsWithActivePins: called with [${dfeNumber}]`)
```

Comment:
you should be OK to remove this now (unless you need it for further debugging?) as the sql service spits out all queries in debug mode

Reply:
@GuyHarwood I didn't write that, I'm assuming whoever wrote it left if for a good reason.

Reply:
unless you need it, please remove it.